### ISSUE TITLE
Visualize lane linking

### ISSUE THREAD

**Issue by johschmitz** [esmini/esmini#391]
I was wondering if a feature could be added to visualize the lane linking when showing the vector data  with the shortcut <kbd>O</kbd>. This would help a lot as a debugging tool during the creation of slightly more complex OpenDRIVE files.

**Comment by eknabevcc** [comment]
https://odrviewer.io 
a perfect tool for OpenDRIVE inspection. After finding that excellent tool, we don't prioritize odr inspection in esmini. 
But for curiosity, what kind of info did you think of and how to visualize? Like a circle/dot indicating that a connection exists perhaps?

**Comment by eknabevcc** [comment]
Ah, now realizing that it would need to be something more, at least for junction where there are potentially multiple connecting lanes from one single lane of incoming road.

**Comment by johschmitz** [comment]
I would draw a little line from one lane to the other for each lane link or something like that and at the end of the line write the lane IDs.

**Comment by eknabevcc** [comment]
I agree that something like this would add value. We'll keep it in mind. Keeping issue open for a while.
